source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -2267,5 +2302,195 @@ namespace RendererQml
 
         return function.str();
     }
+
+	const std::string RendererQml::AdaptiveCardQmlRenderer::getStretchHeight()
```

Comment:
Is it possible to format this function a bit for easy readability?

Reply:
I will try to format these function